lib/generators/components/templates/_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="app-c-<%= @public_name %>">
+  <h2>Please write this component</h2>
```

Comment:
This could include a link to here:
https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#example-yaml-file

"How to build a component"